apps/showcase/src/styles.scss

```diff
@@ -1,28 +1,31 @@
-// Issue with pnp
-// @import "bootstrap/scss/bootstrap";
-// @import "bootstrap-icons/font/bootstrap-icons";
-@import "https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css";
-@import "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css";
+// FIXME sass-loader cannot resolve scss files with pnp
```

Comment:
seems not required anymore

Reply:
we can import `css` files but not `scss` files